backend/app/Http/Controllers/Api/WebsiteController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Api;
+
+use App\Actions\Website\AddWebsiteAction;
+use App\Actions\Website\AddWebsiteRequest;
+use App\Http\Requests\AddWebsiteHttpRequest;
+use App\Http\Resources\WebsiteResource;
+use Illuminate\Http\JsonResponse;
+
+class WebsiteController
+{
+    private $addWebsiteAction;
+
+    public function __construct(AddWebsiteAction $addWebsiteAction)
+    {
+        $this->addWebsiteAction = $addWebsiteAction;
+    }
+
+    public function add(AddWebsiteHttpRequest $request): JsonResponse
+    {
+        $response = $this->addWebsiteAction->execute(
+            AddWebsiteRequest::fromRequest($request)
+        );
+
```

Comment:
For http response use ApiResponse::success method

Reply:
It meanse your response or resource should implement contract ApiResponse

Reply:
Fix